sdk/identity/azure-identity/README.md

```diff
@@ -46,7 +46,7 @@ To take dependency on a particular version of the library that isn't present in
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.11.4</version>
+    <version>1.12.0</version>
```

Comment:
this be 1.12.1 ?

Reply:
No, because this is a dependency version. It just didn't get updated recently (will figure out why.)

Once post-rel runs it will update it to 1.12.1.